app/src/main/java/org/wikipedia/views/WikiTextKeyboardView.java

```diff
@@ -117,12 +117,20 @@ public void setEditText(@NonNull PlainPasteEditText editText) {
         }
         @Nullable String title = null;
         CharSequence selection = editText.getInputConnection().getSelectedText(0);
-        if (selection != null && selection.length() > 0) {
-            title = trimPunctuation(selection.toString()).replace("[[", "").replace("]]", "");
+        String before;
+        String after;
```

Comment:
Just a minor thing that these objects can be moved into line 125.

Reply:
ok!